src/CMakeLists.txt

```diff
@@ -95,9 +95,15 @@ set ( PACKAGE_VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_COMMIT
 ## Find external dependencies.
 find_package(PkgConfig)
 find_package(LibElf REQUIRED)
-find_package(hsakmt 1.0 REQUIRED HINTS ${CMAKE_INSTALL_PREFIX} PATHS /opt/rocm)
 pkg_check_modules(drm REQUIRED IMPORTED_TARGET libdrm)
 
+## Find dependencies that may be built with us or from an installed package.
+if(TARGET hsakmt::hsakmt)
```

Comment:
We should not be checking for `if(TARGET)` as this would require the superproject to order the `add_subdirectory` based on the dependency order. This can be quite fragile as we would need to change the order if the dependencies between components change. Instead we should just let cmake order the dependencies.